api/v1beta1/common_types.go

```diff
@@ -108,6 +113,15 @@ type GlanceAPITemplate struct {
 	APITimeout int `json:"apiTimeout,omitempty"`
 }
 
+// TopologyRef -
+type TopologyRef struct {
+	// +kubebuilder:validation:Optional
+	// Name -
+	Name string `json:"name"`
+	// Namespace -
+	Namespace string `json:"namespace,omitempty"`
```

Comment:
I know that this is how it is used in k8s, just wondering if this provides the impression that we start supporting to spread a deployment across multiple namespaces, which we don't, at least right now. maybe we add that to the description of the parameter. could also add into the defaulting webhook to set it to the namespace of the cr.

Reply:
yeah it makes sense. Both comment and webhooks help here to make sure we don't risk to create false assumptions. I'm going to do that in my next PS.

Reply:
Done, and both `validationCreate` and `validationUpdate` catches a different `namespace`, reporting an error with the associated message (that we do not support customizing the namespace).